src/Components/getList.js

```diff
@@ -60,9 +64,8 @@ const FullList = (props) => {
         </li>
 
         {searchTerm
-           ? searchList.map(item => <GroceryItem item={item} />)
-           : props.data.map(item => <GroceryItem item={item} />)
-        }
+          ? searchList.map(item => <GroceryItem item={item} />)
+          : props.data.map(item => <GroceryItem item={item} />)}
```

Comment:
Same as [this comment](https://github.com/the-collab-lab/tcl-7-smart-shopping-list/pull/32/files?diff=unified&w=1#r439790031), it would be valuable to add the `key` here.